MAINTAINERS

```diff
@@ -3206,7 +3206,8 @@ F:	Documentation/devicetree/bindings/media/rpivid_hevc.jaml
 F:	drivers/staging/media/rpivid
 
 BROADCOM BCM2835 CAMERA DRIVER
-M:	Dave Stevenson <dave.stevenson@raspberrypi.org>
+M:	Dave Stevenson <dave.stevenson@raspberrypi.com>
+M:	Naushir Patuck <naush@raspberrypi.com>
```

Comment:
We do now have kernel-list@raspberrypi.com for the HEVC driver, mainly as we don't want to drop everything on John.
Do we want to use it here instead of adding you?

Reply:
The list of files will also change very soon to be the JAML version of the DT binding.

Reply:
Yes, that seems reasonable.  I've removed both names and added kernel-list@.